### PR TITLE
Update the README to try to be more clear

### PR DESCRIPTION
**Pull request by edwinthinks**
### Describe your change in plain English.

Updates the README to move the setup instructions further to the top with some emojis (because why not!). Moreover, try to streamline the application.yml setup.

### Link to the issue

N/A

### CODE REVIEW

**Review by Thrillberg — APPROVED** [review]
Should we move the Knowledge Base section to the wiki? Otherwise looks great to me!

File: README.md [hunk 1]
```diff
@@ -3,72 +3,70 @@
...
-- email: `b@c.com`
-- password: `123456`
+email: b@c.com
+password: 123456
 
 Staff
-- email: `a@b.com`
-- password: `123456`
+email: a@b.com
+password: 123456
```
Comment (nsiwnf) [inline README.md:52]: I kind of like these on separate lines but not necessary

**Review by nsiwnf — APPROVED** [review]
lgtm
